parserscripts/extract_CRISPRdb.py

```diff
@@ -21,49 +24,59 @@ def get_taxons_from_CRISPRdb():
     dom_homepage = get_dom(url)
     container = dom_homepage('div[class="strainlist"]')[0]
     for link in container('a'):
-        taxon_id = link.href.encode('ascii','ignore')[46:]
+        taxon_id = link.href.encode('ascii', 'ignore')[46:]
         taxon_ids.append(taxon_id)
         with open('taxon_ids.csv', 'a') as csvfile:
             writer = csv.writer(csvfile)
             writer.writerow([taxon_id])
     return taxon_ids
 
+
 def get_sequences(taxon_id):
-    url = "http://crispr.u-psud.fr/cgi-bin/crispr/SpecieProperties.cgi?Taxon_id=" + taxon_id
+    url = ("http://crispr.u-psud.fr/cgi-bin/crispr/SpecieProperties.cgi?"
+           "Taxon_id=" + taxon_id)
     dom = get_dom(url)
     table = dom('table[class="primary_table"]')[1]
     sequences = []
     for sequence in table('tr'):
-        seq = []
-        if sequence('th')==[]:
-            seq = {}
-            seq['Taxon_id'] = taxon_id
-            seq['RefSeq'] = sequence('td')[1].content.encode('ascii','ignore')
+        if not sequence('th'):
+            seq = {
+                'Taxon_id': taxon_id,
+                'RefSeq': sequence('td')[1].content.encode('ascii', 'ignore')
+            }
             sequences.append(seq)
     return sequences
 
+
 def get_loc_ids(sequence):
-    url = "http://crispr.u-psud.fr/crispr/CRISPRProperties.php?RefSeq=" + sequence['RefSeq']
+    url = ("http://crispr.u-psud.fr/crispr/CRISPRProperties.php?RefSeq="
+           + sequence['RefSeq'])
     dom = get_dom(url)
     div = dom('div[class="rightcontent"]')[1]
     table = div('table[class="primary_table"]')[0]
     tbody = table('tbody')[0]
     loc_ids = []
     for crispr_id in tbody('tr'):
         cell = crispr_id('td')[1]
-        crispr_id_value = cell('font')[0].content.encode('ascii','ignore').replace('<br/n/>','')
-        crispr_id_value = crispr_id_value.replace('\n','')
-        crispr_id_value = crispr_id_value.replace('\t','')
-        crispr_id_value = crispr_id_value.replace('<br />','')
+        crispr_id_value = (cell('font')[0]
+                           .content
+                           .encode('ascii', 'ignore')
```

Comment:
@mbonsma do you use this script any more? This doesn't look right. `encode` is `str` -> `bytes` and the next method is working with strings.

Also, `pattern.web` seems broken on Python 3.

Reply:
If we're going to run CRISPRfinder locally, we don't need the scraping part of the script but we'd still need the part that parses the table. (It addresses #35.) 

I'll make a new issue to look this one over.